docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -290,26 +289,23 @@ following steps:
 2. Connect the smartphone to the Wi-Fi Hotspot created in the
    [Configuring Wi-Fi hotspot](#Configuring-a-Wi-Fi-hotspot) section.
 3. Open the CHIPTool application on your smartphone.
-4. Tap the **SCAN QR CODE** button and scan the commissioning QR code.
-5. Tap the **Rendezvous over BLE** button to initiate the commissioning
-   procedure. You will see a few pop-up messages appear as the commissioning
+4. Tap the **PROVISION CHIP DEVICE WITH THREAD** button and scan the commissioning QR code.
+5. You will see a few pop-up messages appear as the commissioning
    progresses. Eventually, the network settings screen appears.
-6. In the new screen, open the **THREAD** tab.
-7. Tap the **Save Network** button to send a Thread provisioning message to the
+6. In the new screen tap the **Save Network** button to send a Thread provisioning message to the
    accessory device.
 
 You will see the "Network provisioning completed" message when the accessory
 device successfully joins the Thread network.
 
 ## Sending CHIP commands
 
-Once the device is commissioned, check the connection with the following
-commands:
+Once the device is commissioned, you should be able to see new window containing
```

Comment:
I think @greg-fer usually prefers "something happens" over "something should happen" in such cases. How about: "Once the device is commissioned, a new screen with a text box and **ON**, **TOGGLE**, **OFF** buttons appears"? Not sure we need to mention the slider if we don't use it below.

Reply:
Partially done, I rephrased it, but I would not remove slider, as it is a part of appearing UI, even if we are not using it.